hack/lib/vars.bash

```diff
@@ -11,13 +11,19 @@ fi
 # shellcheck disable=SC1091,SC1090
 source "$(dirname "${BASH_SOURCE[0]}")/../../test/vendor/knative.dev/test-infra/scripts/e2e-tests.sh"
 
-export KNATIVE_SERVING_VERSION="${KNATIVE_SERVING_VERSION:-v0.15.2}"
-export KNATIVE_EVENTING_VERSION="${KNATIVE_EVENTING_VERSION:-v0.15.2}"
+# Adjust these when upgrading the knative versions.
+readonly KNATIVE_SERVING_VERSION="${KNATIVE_SERVING_VERSION:-v0.15.2}"
```

Comment:
Can we use "export" everywhere rather than "readonly" ? The readonly variables cause problems for example when we want to re-use code from other repos.

Reply:
Thanks for fixing.